mitmproxy/tools/main.py

```diff
@@ -39,15 +39,10 @@ def process_options(parser, opts, args):
     if args.version:
         print(debug.dump_system_info())
         sys.exit(0)
-    if args.options:
-        print(optmanager.dump(opts))
-        sys.exit(0)
-    if args.quiet:
+    if args.quiet or args.options:
```

Comment:
This is just to suppress all other output when we are dumping options (for instance, info on ignored options). The comment would say "# Suppress all other output when dumping options", which is what the code says. ;)